.github/ISSUE_TEMPLATE/NEW_EXPERIMENT.md

```diff
@@ -0,0 +1,86 @@
+---
+name: "\U0001F4CB New experiment proposal"
+about: "Proposal and workflow guideline for adding a new experiment \U0001F52C."
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- ⚠️ If you do not respect this template, your issue will be closed -->
+<!-- ⚠️ Make sure to browse the opened and closed issues -->
+
+# New experiment proposal
+Provide the experiment and implementation details below and ask the qiskit-experiments core team to review 
+your proposal. 
+
+## General details
+
+### Experiment name
+What is the experiment class name? This name will also be used in the API documentation and tutorial.
+
+### Experiment type
+What type the experiment is? Characterization, calibration, verification, validation, else? 
+
+### Experiment protocol
+Provide a concise description of the experiment. Make sure you cover the following aspects:  
+* What is the main goal of the experiment?
+* Describe the circuits/pulses and their main parameters needed for the experiment
+* What are the main outputs of the experiment?
+
+### Experiment analysis
+Provide a concise description of the experiment's analysis. Make sure you cover the following aspects:
+* What is the main fit model for the experiment? 
+* What are the main fit parameters?
```

Comment:
Maybe also add defaults and bounds for the parameters, if relevant?

Reply:
added